src/mon.sh

```diff
@@ -15,16 +15,22 @@ source "$cwd/nginxInfo.sh"
 
 cat > sysinfo.json <<EOF
 {
-        "hostname": "$(hostname)",
-	"cpu-usage": $("$cwd/cpu.sh"),
 EOF
 
-getRAM
-getDiskInfo
-getRoute
-getServices
-getContainers
-getImages
+cat >> sysinfo.json <<EOF
+	"hostname": "$(hostname)",
+	"name": "nginx-info",
+	"project_token": "321084yr1873y4813yr8ewyg3",
```

Comment:
why the token is hardcoded?

Reply:
I do not really understand, how the token works. So, in order to test the scripts, i just put there a random string.